src/app/containers/PageHandlers/withVariant/index.jsx

```diff
@@ -10,7 +10,7 @@ const WithVariant = Component => {
 
     const match = pathOr({}, ['match'], props);
 
-    const redirectPath = getVariantRedirectUrl(match, service, variant);
+    const redirectPath = getVariantRedirectUrl({ ...match, service, variant });
```

Comment:
can we be explicit in what we're passing down? e.g.

```js
const { path, params } = pathOr({}, ['match'], props);

const redirectPath = getVariantRedirectUrl({ path, params, service, variant });
```

Reply:
Fixed!